hack/bazel-push-images.sh

```diff
@@ -29,31 +29,41 @@ default_targets="
     virt-controller
     virt-handler
     virt-launcher
-    virt-exportserver
-    virt-exportproxy
-    conformance
-    libguestfs-tools
-    pr-helper
-    example-hook-sidecar
-    example-disk-mutation-hook-sidecar
-    example-cloudinit-hook-sidecar
     alpine-container-disk-demo
-    cirros-container-disk-demo
-    cirros-custom-container-disk-demo
-    virtio-container-disk
-    alpine-ext-kernel-boot-demo
     fedora-with-test-tooling-container-disk
-    alpine-with-test-tooling-container-disk
-    fedora-realtime-container-disk
-    disks-images-provider
-    nfs-server
     vm-killer
-    winrmcli
     sidecar-shim
-    network-slirp-binding
-    network-passt-binding
 "
 
+case ${ARCHITECTURE} in
+"s390x" | "crossbuild-s390x") ;;
+#"aarch64" | "crossbuild-aarch64")
```

Comment:
@jschintag is this a leftover or is this something we want to enable?

Reply:
I mainly put it there in case there was something special we want to do with arm, but i can remove it as well.

Reply:
Lets remove it for now just to avoid confusion.

Reply:
Done